templates/lock.mdx

```diff
@@ -2,11 +2,11 @@
 title: lock template
 ---
 
-## Description
+> In-game description of the lock, typically from sys.upgrades { full: true }
 
-In-game description of the lock, typically from sys.upgrades { full: true }
+<lock> is a tier <N> lock {created by the <in-game corp> corporation | exclude if the lock does not announce its manufacturer}.
 
-### Subheader for stats
+### Stats
 
 stats?
```

Comment:
Clarify this.

Reply:
Should be resolved by [fc94ee4](https://github.com/comcode-org/hackmud_wiki/pull/116/commits/fc94ee4512c5d3650d7fc00f718833aec736878f)